src/lib/PlaylistCards.svelte

```diff
@@ -96,4 +98,18 @@
       opacity: 1;
     }
   }
+
+  .overridden {
+    animation: colorTransition 3s linear infinite alternate;
+  }
+
+  @keyframes colorTransition {
+    0%,
+    100% {
+      box-shadow: 0 0 15px 5px rgba(255, 215, 0, 0.4);
+    }
+    50% {
+      box-shadow: 0 0 15px 5px rgba(255, 217, 0, 0.635);
+    }
```

Comment:
Sorry - I think I am happy with this now. Changes transition to 5s and brings in the border a bit
```
  .overridden {
    animation: colorTransition 5s linear infinite alternate;
  }

  @keyframes colorTransition {
    0%,
    100% {
      box-shadow: 0 0 7px 3px rgba(255, 215, 0, 0.4);
    }
    50% {
      box-shadow: 0 0 7px 3px rgba(255, 217, 0, 0.635);
    }